### PR TITLE
Change old method to new method, to pass compile file.

### PR DESCRIPTION
**Pull request by hanhan9449**
Change `get_local` and `set_local` to `local.get` and `local.set`, to pass compile file.

### Description

Change `get_local` and `set_local` to `local.get` and `local.set`, to pass compile file.

### CODE REVIEW

**Comment by hanhan9449** [comment]
Sync with English version

File: files/zh-cn/webassembly/understanding_the_text_format/index.md [hunk 1]
```diff
@@ -83,43 +83,43 @@ WebAssembly 模块中的所有代码都是划分到函数里面。函数具有
 
 ## 获取和设置局部变量和参数
 
-局部变量和参数能够被函数体使用 get_local 和 set_local 指令进行读写。
+局部变量和参数能够被函数体使用 local.get 和 local.set 指令进行读写。
```
Comment (jasonren0403) [inline files/zh-cn/webassembly/understanding_the_text_format/index.md:86]: Let's also add some code styles

```suggestion
局部变量和参数能够被函数体使用 `local.get` 和 `local.set` 指令进行读写。
```

File: files/zh-cn/webassembly/understanding_the_text_format/index.md [hunk 2]
```diff
@@ -83,43 +83,43 @@ WebAssembly 模块中的所有代码都是划分到函数里面。函数具有
...
 
-get_local/set_local 指令使用数字索引来指向将被存取的条目：按照它们的声明顺序，参数在前，局部变量在后。因此，给定下面的函数：
+local.get/local.set 指令使用数字索引来指向将被存取的条目：按照它们的声明顺序，参数在前，局部变量在后。因此，给定下面的函数：
```
Comment (jasonren0403) [inline files/zh-cn/webassembly/understanding_the_text_format/index.md:88]:
```suggestion
`local.get`/`local.set` 指令使用数字索引来指向将被存取的条目：按照它们的声明顺序，参数在前，局部变量在后。因此，给定下面的函数：
```

**Review by jasonren0403 — COMMENTED** [review]
👍 nice to have them, beware of the code styles

File: files/zh-cn/webassembly/understanding_the_text_format/index.md [hunk 3]
```diff
@@ -83,43 +83,43 @@ WebAssembly 模块中的所有代码都是划分到函数里面。函数具有
...
+  local.get 1
+  local.get 2)
 ```
 
-- get_local 0 会得到 i32 类型的参数
-- get_local 1 会得到 f32 类型的参数
-- get_local 2 会得到 f64 类型的局部变量
+- local.get 0 会得到 i32 类型的参数
+- local.get 1 会得到 f32 类型的参数
+- local.get 2 会得到 f64 类型的局部变量
```
Comment (jasonren0403) [inline files/zh-cn/webassembly/understanding_the_text_format/index.md:99]:
```suggestion
- `local.get 0` 会得到 i32 类型的参数
- `local.get 1` 会得到 f32 类型的参数
- `local.get 2` 会得到 f64 类型的局部变量
```

File: files/zh-cn/webassembly/understanding_the_text_format/index.md [hunk 4]
```diff
@@ -83,43 +83,43 @@ WebAssembly 模块中的所有代码都是划分到函数里面。函数具有
...
 
 由于使用数字索引来指向某个条目容易让人混淆，因此，也可以通过别名的方式来访问它们，方法就是在类型声明的前面添加一个使用美元符号（$）作为前缀的名字。例如：
 
 ```wasm
 (func (param $p1 i32) (param $p2 f32) (local $loc i32) …)
 ```
 
-这里，使用 get_local $p1 就代替 get_local 0，访问参数 i32 变量时，就可以通过 $p1 进行访问。
+这里，使用 local.get $p1 就代替 local.get 0，访问参数 i32 变量时，就可以通过 $p1 进行访问。
```
Comment (jasonren0403) [inline files/zh-cn/webassembly/understanding_the_text_format/index.md:107]:
```suggestion
这里，使用 `local.get $p1` 就可以代替 `local.get 0`，访问参数 i32 变量时，就可以通过 $p1 进行访问。
```

File: files/zh-cn/webassembly/understanding_the_text_format/index.md [hunk 5]
```diff
@@ -83,43 +83,43 @@ WebAssembly 模块中的所有代码都是划分到函数里面。函数具有
...
 
 > **备注：** 当文本转换为二进制后，二进制中只包含整数。
 
 ## 栈式机器
 
 虽然浏览器把 wasm 编译为某种更高效的东西，但是，wasm 的执行是以栈式机器定义的。也就是说，其基本理念是每种类型的指令都是在栈上执行数值的入栈出栈操作。
 
-例如，get_local 被定义为把它读到的局部变量值压入到栈上，然后 i32.add 从栈上取出两个 i32 类型值（它的含义是把前面压入栈上的两个值取出来）计算它们的和（以 2^32 求模），最后把结果压入栈上。
+例如，local.get 被定义为把它读到的局部变量值压入到栈上，然后 i32.add 从栈上取出两个 i32 类型值（它的含义是把前面压入栈上的两个值取出来）计算它们的和（以 2^32 求模），最后把结果压入栈上。
```
Comment (jasonren0403) [inline files/zh-cn/webassembly/understanding_the_text_format/index.md:115]:
```suggestion
例如，`local.get` 被定义为把它读到的局部变量值压入到栈上，然后 `i32.add` 从栈上取出两个 `i32` 类型值（它的含义是把前面压入栈上的两个值取出来）计算它们的和（以 2^32 求模），最后把结果压入栈上。
```

**Review by jasonren0403 — APPROVED** [review]
👍

**Pull request merged**